docs/guides/1.20.0-upgrade-guide.md

```diff
@@ -19,6 +19,26 @@ The Terraform MongoDB Atlas Provider version 1.20.0 has a number of new and exci
   - data-source/mongodbatlas_data_lake_pipeline_run
   - data-source/mongodbatlas_data_lake_pipeline_runs
 
+**Breaking Changes:**
+- `mongodbatlas_alert_configuration` resource attribute `notification.#.integration_id` may have a computed value due to recent updates in the Atlas API [Alert Configuration](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Alert-Configurations/operation/getAlertConfiguration). Due to this MongoDB Atlas Provider versions **1.16.0 to 1.19.0** may experience non-empty plans if using this resource with an integration set for the notifications without an explicitly set `integration_id`. For more details, see [#2603](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2603).
+
+  These users should either:
+
+  - Upgrade to this latest MongoDB Atlas Terraform Provider (1.20.0) with the required updates to avoid the non-empty plan, or
```

Comment:
what do you mean by `with the required updates`?

Reply:
In case you mean the updates necessary to update to 1.20, I would not mention it:
`Upgrade to this latest MongoDB Atlas Terraform Provider (1.20.0) to avoid the non-empty plan, or`
to keep it simple and to the point.